sui-execution/latest/sui-verifier/src/private_generics_verifier_v2.rs

```diff
@@ -16,10 +16,15 @@ use sui_types::{
 
```

Comment:
You now only need to change `FUNCTIONS_TO_CHECK`, hooray! But you still need to update the compiler as well